spec/stubbable_endpoint/app.rb

```diff
@@ -33,8 +33,11 @@ def response_body
           instances: 2,
           state: "STARTED",
           environment_json: {},
+          space_url: "/v2/spaces/example-space-guid",
         },
-        route: "route-1"
+        route: "route-1",
+        space: "test-space-name",
```

Comment:
are these needed in the test data? do the accurately reflect what the cloud foundry API returns?